be/src/exprs/agg/count.h

```diff
@@ -70,13 +70,8 @@ class CountAggregateFunction final
         down_cast<Int64Column*>(to)->append(this->data(state).count);
     }
 
-    void batch_finalize(size_t chunk_size, const Buffer<AggDataPtr>& agg_states, size_t state_offsets,
```

Comment:
Why remove `batch_finalize` methods?

Reply:
the interface is 
```
void batch_finalize(FunctionContext* ctx, size_t chunk_size, const Buffer<AggDataPtr>& agg_states,
                        size_t state_offset, Column* to) const
```

So it has not been called before.

Reply:
After changing to override, the actual test has no effect on performance, perhaps The function is actually inlined

Reply:
It should be batch_finalize interface changed, but not updated the implementation in `count.h`, `sum.h`.